src/ansys/geometry/core/designer/component.py

```diff
@@ -1171,6 +1224,94 @@ def create_beams(
         self._beams.extend(new_beams)
         return self._beams[-n_beams:]
 
+    def __create_beams(
+        self,
+        segments: list[tuple[Point3D, Point3D]],
+        profile: BeamProfile,
+        arcs: list[Arc] = None,
+        circles: list[Circle] = None,
```

Comment:
```suggestion
        arcs: list[Arc],
        circles: list[Circle],
```
Private methods should not have default values IMO - only developers are expected to call them and we should always know what we are passing in